packages/@okta/vuepress-site/docs/reference/okta-expression-language-in-oie/index.md

```diff
@@ -0,0 +1,247 @@
+| `$zoned_date_time_object.minusSeconds`   | (int seconds)                        | ZonedDateTime   | `Time.now().minusSeconds(2)`                                          | 2015-07-31T17:18:35.979Z                                                    |
+| `$zoned_date_time_object.withinDays`     | (int days)                           | Boolean         | `user.created.withinDays(1)`                                          | True                                                                        |
+| `$zoned_date_time_object.withinHours`    | (int hours)                          | Boolean         | `user.created.withinHours(100)`                                       | True                                                                        |
+| `$zoned_date_time_object.withinMinutes`  | (int minutes)                        | Boolean         | `user.created.withinMinutes(2)`                                       | False                                                                       |
+| `$zoned_date_time_object.withinSeconds`  | (int seconds)                        | Boolean         | `user.created.withinSeconds(100)`                                     | False                                                                       |
+
+>Note: Okta supports the use of the time zone IDs and aliases listed in [the Time Zone Codes table](/docs/reference/okta-expression-language/#appendix-time-zone-codes).
+
+## Constants and Operators
+
+| Common Action                                                                               | Example                                                     |
+| ----------------                                                                            | --------                                                    |
+| Refer to a `String` constant                                                                | `'Hello world'`                                             |
+| Refer to a `Integer` constant                                                               | `1234`                                                      |
+| Refer to a `Number` constant                                                                | `3.141`                                                     |
+| Refer to a `Boolean` constant                                                               | `true`                                                      |
+| Concatenate two strings                                                                     | `user.profile.firstName + user.profile.lastName`            |
+
+## Conditional Expressions
+
+<br>There are several rules for specifying the condition.
+
+* Expressions must have valid syntax.
+* Expressions must evaluate to Boolean.
+* Expressions cannot contain an assignment operator, such as =.
+* User attributes used in expressions can contain only available User attributes.
```

Comment:
I'm sorry, I'm not following.  What does this mean?

I think it means something like `User attributes referenced in an expression must exist`?